.github/workflows/cicd.yml

```
@@ -0,0 +1,44 @@
+              uses: dorny/paths-filter@v3
+              with:
+                  filters: |
+                      api:
+                        - 'packages/api/**'
+
+            - name: Deploy API
+              if: ${{ steps.changes.outputs.api == 'true' }}
+              uses: ./.github/workflows/cd.yml
+
+        outputs:
+            api_changed: ${{ steps.paths.outputs.api }}
```

actionlint (1.7.4)

property "paths" is not defined in object type {changes: {conclusion: string; outcome: string; outputs: object}}

(expression)